lms/djangoapps/discussion_api/api.py

```diff
@@ -782,7 +782,8 @@ def _handle_read_field(api_content, form_value, user, cc_content):
     """
     if form_value and not cc_content['read']:
         user.read(cc_content)
-        api_content["unread_comment_count"] -= 1
+        # When a thread is marked as read, it is supposed all its comments are read
```

Comment:
@wajeeha-khalid can we please update this comment to add information about the responses as well? something like, 
```
When a thread is marked as read, all of its responses and comments are also marked as read.
```